Upgrading guide - finishing touches (#1795)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

Use the following procedures to clone your {Project} instances to preserve your environments in preparation for upgrade.

The {Project} clone tool does not support migrating a {SmartProxyServer} to {EL} 8.

> Instead you must backup the existing {SmartProxyServer}, restore it on {EL} 8, then reconfigure {SmartProxyServer}.

Author: I do not know whether the procedure in this file is the same for EL 8. Is there someone who could confirm this?

Reviewer: "Cloning to the Target Server", step 4. should probably enable RHEL8 packages. Otherwise I wonder if all stays the same, I'm trying to check.

Author: Thanks so much for checking! I'll ask in the satellite IRC channel, too

> Instead you must backup the existing {SmartProxyServer}, restore it on {EL} 8, then reconfigure {SmartProxyServer}.

Reviewer 2: ```suggestion
Instead, you must backup the existing {SmartProxyServer}, restore it on {EL} 8, and then reconfigure {SmartProxyServer}.
```

To clone {ProjectServer}, ensure that you have the following resources available:

* A minimal install of {EL} 8 to become the target server.

> Do not install {EL} 8 software groups, or third-party applications.

Reviewer 2: ```suggestion
Do not install {EL} 8 software groups or third-party applications.
```

> Ensure that your server complies with all the specifications of {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.

Reviewer 2: Alternative:

```suggestion
Ensure that your server complies with all the required specifications.
For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.
```

I am unsure if my suggestions adds a benefit apart from consistency (which is a benefit in itself). Your call.
